Userland/Utilities/hexdump.cpp

```diff
@@ -67,43 +63,42 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         putchar('\n');
     };
 
-    Array<u8, BUFSIZ> contents;
-    Span<u8> previous_line;
-    static_assert(LINE_LENGTH_BYTES * 2 <= contents.size(), "Buffer is too small?!");
-    size_t bytes_in_buffer = 0;
+    Array<u8, BUFSIZ> buffer;
+    Span<u8> bytes = {};
+    Span<u8> previous_line = {};
```

Comment:
This kind of declaration will create a temporary before copying it to `previous_line`.
No need to do all that job, `Span<u8> previous_line;` is fine.

Reply:
The same applies to `bytes`.
You can also use these aliases:
```cpp
using ReadonlyBytes = Span<u8 const>;
using Bytes = Span<u8>;
```

Reply:
On line 75, I rely on `bytes.size()` being 0 the first time through the loop: `auto bytes_to_read = contents.size() - bytes.size();`

Will this check still work if I skip the initialization?

Reply:
You did not skip the initialization, the default constructor is called anyway and both field of `Span` are properly initialized in the class definition

Reply:
Got it. I've removed the `= {}` from both variables, and learned something new about C++